compiler/rustc_codegen_ssa/src/base.rs

```diff
@@ -735,6 +735,11 @@ pub fn codegen_crate<B: ExtraBackendMethods>(
     crate::assert_module_sources::assert_module_sources(tcx, &|cgu_reuse_tracker| {
         for (i, cgu) in codegen_units.iter().enumerate() {
             let cgu_reuse = cgu_reuse[i];
+
+            if tcx.sess.opts.unstable_opts.print_mono_items {
+                println!("CGU_REUSE {} {cgu_reuse}", cgu.name());
```

Comment:
Maybe do this inside the cgu_reuse_tracker instead? The current changes in this PR don't apply to cg_clif.